packages/pspy/PKGBUILD

```diff
@@ -1,39 +1,55 @@
 # This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
 # See COPYING for license details.
 
+# ArchLinux go guidelines --
+# https://wiki.archlinux.org/index.php/Go_package_guidelines
+
 pkgname=pspy
 pkgver=159.2312eed
 pkgrel=1
 pkgdesc='Monitor linux processes without root permissions.'
 arch=('x86_64' 'aarch64')
 groups=('blackarch' 'blackarch-misc' 'blackarch-recon')
-url='https://github.com/DominicBreuker/pspy'
+url="https://github.com/DominicBreuker/$pkgname"
 license=('GPL3')
-depends=('glibc')
 makedepends=('git' 'go-pie')
-source=("git+https://github.com/DominicBreuker/$pkgname.git")
-sha512sums=('SKIP')
+source=("$url/archive/master.zip")
+sha256sums=('7774cc9df3bbf7def59b5ab6ca97b048c40ccd155f9bb0fe9d13ecd42c217725')
 
-pkgver() {
-  cd $pkgname
+prepare(){
+    mkdir -p build/
 
-  echo $(git rev-list --count HEAD).$(git rev-parse --short HEAD)
+    VERSION="v1.2.0-10-g2312eed"
```

Comment:
You need to keep `echo $(git rev-list --count HEAD).$(git rev-parse --short HEAD)`, it's for automatic pkgver update. Don't hardcode the actual version in here.